mantidproject/mantid
# Warnings about missing pip packages

**Description of work**
This PR does a couple of things:
- It creates a utilities file for checking if a pip package is already installed. This ensures a consistent warning message is displayed if a pip package is not installed
- It adds a user-facing documentation page on how to do a pip install in a mantid conda environment

**To test:**
Read the PipInstall instructions and make sure links work.

**Test A** (on Windows): 

Checkout this branch and build
Open Workbench
Open `Indirect`->`Bayes`
Go to Quasi tab
Load the `iris26176` file ending in `_red` found in the Sample test data 
Load the same file for the resolution
Click run and there should be a warning message with a link to the pip install instructions.
Close Workbench

**Test B** (On IDAaaS)
Attempt the pip install instructions for the nightly

Open `Indirect`->`Bayes`
Go to Quasi tab
Load the `iris26176` file ending in `_red` found in the Sample test data 
Load the same file for the resolution
Click run and make sure it runs successfully

Part of #36152

#### Reviewer ####

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

#### Gatekeeper ####

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
